owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -538,6 +543,12 @@ interface FileDao {
             WHERE parentId = :folderId AND mimeType LIKE :mimeType || '%'
         """
 
+        private const val SELECT_FILES_FOR_ACCOUNT = """
```

Comment:
Adapt the name as well:
```suggestion
        private const val SELECT_DOWNLOADED_FILES_FOR_ACCOUNT = """
```

Reply:
Done